TYPO3.Media/Documentation/References/ViewHelpers.rst

```diff
@@ -11,9 +11,9 @@ This reference was automatically generated from code on 2016-06-07
 typo3.media:fileTypeIcon
 ------------------------
 
-Renders an <img> HTML tag for a filetype icon for a given TYPO3.Media's asset instance
+Renders an <img> HTML tag for a filetype icon for a given Neos.Media's asset instance
```

Comment:
The references are generated automatically, they should not be edit manually anyways…